File: day Two/java/afterbreak/AverageScore.java
public class AverageScore{

    public static void averageScore(int Score1, int Score2, int Score3){

    int average = (Score1 + Score2 + Score3) / 3;

    if(average >= 90 && average <= 100){
        System.out.println("average is 'A'");    
    }

    else if(average >= 80 && average < 90){
        System.out.println("average is 'B'");    
    }

    else if(average >= 70 && average < 80){
        System.out.println("average is 'C'");    
    }

    else if(average >= 60 && average < 70){
        System.out.println("average is 'D'");    
    }

    else if(average >= 0 && average <= 60){
        System.out.println("average is 'F'");    
    }


}


  public static void main(String[] args){
   
    
     int Score1 = 60;
     int Score2 = 95;
     int Score3 = 85;
    
    averageScore(Score1, Score2, Score3);  

    }

}

File: day Two/python/afterbreak/AverageScore.py
def AverageScore(Score1, Score2, Score3):

    average = (Score1 + Score2 + Score3) / 3;

    if(average >= 90 and average <= 100):
            print("average is 'A'");    
        
    elif(average >= 80 and average < 90):
            print("average is 'B'");    
        
    elif(average >= 70 and average < 80):
            print("average is 'C'");    
        
    elif(average >= 60 and average < 70):
            print("average is 'D'");    
        
    elif(average >= 0 and average <= 60):
           print("average is 'F'");    
        

   
AverageScore(60, 95, 85);  
    
